packages/nouns-contracts/yarn.lock

```diff
@@ -1,8687 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why was this deleted?

Reply:
As discussed on Discord, locally yarn.lock and got updated and it resulted in the Netlify errors. So this is actually a copy from the master yarn.lock to revert and fix the errors